horace_core/sqw/@sqw/private/cut_accumulate_data_.m

```diff
@@ -241,7 +241,7 @@
         % PageOp_sqw_join operation.
         pix_comb_info = cut_data_from_file_job.accumulate_pix(pix_comb_info, false, ...
             pix_ok, pix_indx, npix, ...
-            buf_size);
+            buf_size,ll);
```

Comment:
worth stating what 11 represents

Reply:
well, it is described and your requessting to modify the description wording in the next comment